baybe/acquisition/base.py

```diff
@@ -39,15 +39,19 @@ def to_botorch(
         self,
         surrogate: Surrogate,
         searchspace: SearchSpace,
-        train_x: pd.DataFrame,
-        train_y: pd.DataFrame,
+        measurements: pd.DataFrame,
```

Comment:
I can already "feel" that `objective` will probably become part of the signature as well, i.e. simply because you usually need it wherever `searchspace` is needed. Look, for instance, at where this function is called, which sort of implies the same:
<img width="558" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/b9dfe29c-9529-4a01-b1d7-2a9a1af31ef4">
Right now, it's not yet needed, but I guess the latest when we go to multi-object, the following line below will need it:
<img width="474" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/5354e84c-cddd-4572-a92c-492dab324d6e">
What are your thoughts?

Reply:
i agree and think the same but theres nothing to discuss because well simply add it when it becomes necessary

Reply:
alright, resolving then